### DISCUSSION TITLE
XTTS v2, can reduce model size by removing languages from model?

### DISCUSSION THREAD

**Discussion by kunibald413** [oobabooga/text-generation-webui#4835] [Q&A] [answered]
Can the size of the model be reduced by removing languages? If so any tips on how to go about it and how much it would cut? I'm only interested in keeping english.

**Comment by TheLounger** [comment, marked as answer]
Modifying/creating models would be more suitable to discuss at [coqui-ai/TTS](https://github.com/coqui-ai/TTS/discussions).
The `TTS` package has the ability to [load other models](https://github.com/coqui-ai/TTS/blob/c99e885cc8a35639ca124a6cad9573bb23f06764/TTS/api.py#L214) but webui just [requests](https://github.com/oobabooga/text-generation-webui/blob/2694ef45a36b98efa1cb81ff2989c4eaf5d27c00/extensions/coqui_tts/script.py#L37) the default one and lets it download it.
